CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# v1.2.4
+
+## Fix build
```

Comment:
The previous version didn't get released, so no need to bump — it's not changing the lib, right?

Reply:
It did: https://www.npmjs.com/package/release-to-github-with-changelog

Reply:
Indeed, only the github changelog bit broke, so we need to repush a new version too :(

Reply:
Oh crap!